Makefile

```diff
@@ -47,6 +47,7 @@ provider.%.build:	provider.%.generate
 
 # Example: make provider.random.generate
 provider.%.generate:	bin/pulumi-java-gen
+	rm -rf providers/pulumi-$*/sdk/
```

Comment:
./bin/pulumi-java-gen does that already in https://github.com/pulumi/pulumi-java/blob/main/pkg/cmd/pulumi-java-gen/main.go#L153 so perhaps no need here.